Centralized configuration - part 1

bobo-k2 opened the pull request:
**Pull Request Summary**

This is the first PR to centralize the portal configuration and simply enable/disable certain features, like bridges or XCM

Configuration is stored in `fetures.ts` and settings looks like
```
// Bridges
export const nativeBridgeEnabled = true;
export const layerZeroBridgeEnabled = true;
export const layerSwapBridgeEnabled = true;
export const celerBridgeEnabled = true;
export const omniBridgeEnabled = true;
export const stargateBridgeEnabled = true;
export const stakeStoneBridgeEnabled = true;
export const arthSwapBridgeEnabled = true;
```

The portal code is updated to take those settings into consideration. 
I found that disabling `router-link`, so I created replacement component in `CustomRouterLink.vue`

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

impelcrypto commented:
@bobo-k2 
I think we can remove this variable too.
https://github.com/AstarNetwork/astar-apps/blob/feat/central-config/src/hooks/useNetworkInfo.ts#L115

bobo-k2 commented:
> @bobo-k2 I think we can remove this variable too. https://github.com/AstarNetwork/astar-apps/blob/feat/central-config/src/hooks/useNetworkInfo.ts#L115

You are right, I updated the variable to return to false, but forgot to remove it. Fixed in https://github.com/AstarNetwork/astar-apps/pull/1362/commits/7a974f828d52e00a14398d335fbe8741da5944ed

bobo-k2 commented on src/features.ts:
```diff
@@ -0,0 +1,27 @@
+export const layerSwapBridgeEnabled = true;
+export const celerBridgeEnabled = true;
+export const omniBridgeEnabled = true;
+export const stargateBridgeEnabled = true;
```
Yes, variables can be private since they are used only by `isBridgeEnabled` function
Fixed in https://github.com/AstarNetwork/astar-apps/pull/1362/commits/060d2813b258aae31a1cd81b69382775dd4b6b7f

gtg7784 reviewed (approved):
Thanks
